tests/test_admin_logs.py

```
@@ -0,0 +1,117 @@
+from app.core.settings import Settings
+
+
+def _make_settings(tmp_path: str) -> Settings:
+    # Construct Settings overriding LOG_DIR and ADMIN_API_KEY for tests
+    s = Settings(LOG_DIR=str(tmp_path), ADMIN_API_KEY="testkey")
+    return s
+
+
+def _write_lines(path: str, lines):
+    with open(path, "w", encoding="utf-8") as fh:
+        for l in lines:
```

Ruff (0.13.1)

Ambiguous variable name: `l`

(E741)